marge/batch_job.py

```diff
@@ -203,6 +220,11 @@ def execute(self):
         self._repo.checkout_branch(target_branch, 'origin/%s' % target_branch)
         self._repo.checkout_branch(BatchMergeJob.BATCH_BRANCH_NAME, 'origin/%s' % target_branch)
 
+        batch_mr = self.create_batch_mr(
```

Comment:
This was somewhat surprising, because so far the batch MR creation has been guarded by `only_allow_merge_if_pipeline_succeeds` flag. Doing it here earlier works as well: if run without your new config options it's going to be used in the older sequence as expected.

Reply:
Yep